content/logo/index.mdx

```diff
@@ -1,6 +1,10 @@
 ---
 seo:
   title: SSW Logos
+  description: >-
+    Discover SSW's brand logos: official designs that symbolize our tech
+    expertise and consulting prowess. Ideal for media use.
+  canonical: 'https://ssw.com.au/logos'
```

Comment:
the route for this page is `/logo`:
![image](https://github.com/SSWConsulting/SSW.Website/assets/87786621/0335af31-273e-463e-a894-6e9b7370aad2)
**Figure: screenshot of logo page**

Should the canonical be `https://ssw.com.au/logo`? (singular logo vs pluralised)